src/pages/library/datasetBuilder/CohortEditor.test.ts

```diff
@@ -178,8 +178,12 @@ describe('CohortEditor', () => {
       updateCriteria,
     });
     // Act
-    screen.getByLabelText(`${criteria.name} low slider`);
-    screen.getByLabelText(`${criteria.name} high slider`);
+    fireEvent.keyDown(screen.getByLabelText(`${criteria.name} low slider`), { keyCode: 39 /* Right Arrow */ });
+    fireEvent.keyDown(screen.getByLabelText(`${criteria.name} high slider`), { keyCode: 37 /* Left Arrow */ });
+
+    // Arrange
+    expect(updateCriteria).toBeCalledWith({ ...criteria, low: 56 });
+    expect(updateCriteria).toBeCalledWith({ ...criteria, high: 98 });
```

Comment:
If you had local `const`s for `low` and `high` you could write
```suggestion
    const low = 55
    const high = 99
    ...
     const criteria = criteriaFromOption({
      id: 0,
      name: 'range',
      kind: 'range',
      min:low,
      max:high,
    });
    ...
    expect(updateCriteria).toBeCalledWith({ ...criteria, low: low +1 });
    expect(updateCriteria).toBeCalledWith({ ...criteria, high: high - 1 });
```